styles.css

```diff
@@ -0,0 +1,1916 @@
+    height: 48px;
+    left: 9px;
+    bottom: 4px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    color: #396df2;
+    text-align: center;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-top: 30px;
+    margin-left: -160px;
+    background-color: #fff;
+  }
+
+  .mobile-modal-fader {
+    margin-top: -50px;
+    margin-left: -73px;
+  }
+
+  #github-mobile {
+    width: 17.99px;
+    height: 20px;
+    margin-left: 250px;
+    margin-top: -100px;
+  }
+
+  .mobile-seesource {
+    width: 136px;
+    height: 48px;
+    left: 9px;
+    bottom: 4px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    background-color: #fff;
+    color: #396df2;
+    text-align: center;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-top: -53px;
+    margin-bottom: 25px;
+    margin-left: 150px;
+  }
+
+  /* MOBILE CARD 2 */
+
+  .mobile-modal1 {
+    position: fixed; /* Poner propiedad de blur aca */
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    background-color: #111111bd;
+    display: flex;
+    opacity: 0;
+    pointer-events: none;
+    transition: opacity 0.6s;
+  }
+
+  .mobile-modal--show1 {
+    opacity: 1;
+    pointer-events: unset;
+    transition: opacity 0.6s;
+  }
+
+  .title-modal-mobile1 {
+    max-width: 90%;
+    font-weight: 700;
+    font-size: 32px;
+    line-height: 44px;
+    color: #172b4d;
+    margin-top: 15px;
+    margin-left: -135px;
+  }
+
+  .mobile-modal-close1 {
+    text-decoration: none;
+    margin-left: 280px;
+    margin-top: -1100px;
+    transition: background-color 0.3s;
+  }
+
+  .mobile-modal-close1:hover {
+    background-color: #6070ff;
+  }
+
+  #modal-mobile-canopy1 {
+    list-style-type: none;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #344563;
+    margin-left: -165px;
+    margin-top: -45px;
+  }
+
+  .mobile-punto-modal01 {
+    list-style-type: none;
+    margin-top: -35px;
+    margin-left: -64px;
+  }
+
+  .mobile-punto-modal02 {
+    list-style-type: none;
+    margin-top: -35px;
+    margin-left: 42px;
+  }
+
+  .mobile-modal-año01 {
+    list-style-type: none;
+    font-weight: 600;
+    font-size: 13px;
+    line-height: 16px;
+    color: #7a869a;
+    margin-top: -31px;
+    margin-left: 55px;
+  }
+
+  .mobile-modal-html01 {
+    margin-left: -255px;
+  }
+
+  .mobile-modal-css01 {
+    margin-left: -135px;
+    margin-top: -25px;
+  }
+
+  .mobile-modal-javascript01 {
+    margin-left: 22px;
+    margin-top: -24px;
+    margin-bottom: 20px;
+  }
+
+  .mobile-seelive01 {
+    width: 136px;
+    height: 48px;
+    left: 9px;
+    bottom: 4px;
+    border: 1px solid #6070ff;
+    border-radius: 8px;
+    color: #396df2;
+    text-align: center;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    margin-top: 20px;
+    margin-bottom: 20px;
+    margin-left: -160px;
+    background-color: #fff;
+  }
+
+  .mobile-seesource01 {
```

Comment:
- [ ] Great job getting the mobile popup right. At the moment your See Live and See Source buttons text are not well aligned as the figma design. Kindly get the alignment to be exact as that of figma.
- Your design
![image](https://github.com/maximoortelli/Portfolio-reactivo/assets/62849872/0384d8d0-89e0-4831-b987-808a1af7be26)
- Figma design
![image](https://github.com/maximoortelli/Portfolio-reactivo/assets/62849872/341b068a-dd13-40bd-8748-bae594c0ac94)